### PR TITLE
Improve xmr proof service

### PR DESCRIPTION
**Pull request by chimp1984**
Fixes some issues brought up by [kayabanerve](https://github.com/kayabaNerve)

### CODE REVIEW

**Comment by ripcurlx** [comment]
@jmacxx Could you please review this PR? Thanks!

**Comment by ghost** [comment]
@ripcurlx yes, I am in process of reviewing/testing the code.  Will let you know ASAP.

This popup text could probably use some revision from yourself / @m52go / @w0000000t to make it more understandable.   It is shown at startup (only users with XMR accounts).

image

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -1534,7 +1534,10 @@ setting.info.msg=When selling BTC for XMR you can use the auto-confirm feature t
   You can also set the maximum amount of BTC per trade to auto-confirm as well as the number of required \
   confirmations here in Settings.\n\n\
   See more details (including how to set up your own explorer node) on the Bisq wiki \
-  [HYPERLINK:https://bisq.wiki/Trading_Monero#Auto-confirming_trades]
+  [HYPERLINK:https://bisq.wiki/Trading_Monero#Auto-confirming_trades]\n\n\
+  Please be aware that when changing the Bisq data directory you must change the Monero wallet and address to avoid \
+  vulnerabilities related to transaction key re-use. More details can be found here: [HYPERLINK:https://web.getmonero.org/2018/09/25/a-post-mortum-of-the-burning-bug.html].
```
Comment (erciccione) [inline core/src/main/resources/i18n/displayStrings.properties]:
```suggestion
  vulnerabilities related to transaction key re-use. More details can be found here: [HYPERLINK:https://www.getmonero.org/2018/09/25/a-post-mortum-of-the-burning-bug.html].
```
Super minor, but please use `www`, which is the canonical address. Using `web` would result in a cache skip.

**Review by ghost — APPROVED** [review]
ACK
I tested mainly the subaddress generation - works fine for me.

**Review by ripcurlx — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/6217#pullrequestreview-991349219

**Comment by w0000000t** [comment]
> @ripcurlx yes, I am in process of reviewing/testing the code. Will let you know ASAP.
> 
> This popup text could probably use some revision from yourself / @m52go / @w0000000t to make it more understandable. It is shown at startup (only users with XMR accounts).
> 
> image

I thought I had replied already, sorry for the delay! I probably left the browser tab hanging and it went lost.
What I wrote the last time was to make the popup text shorter and more effective, like so:

```
WARNING
After changing data directory you must also change XMR wallet and address for security reasons [1]
```

**Comment by ghost** [comment]
@w0000000t thanks that looks good to me.  How about this revision:


> **Notice for XMR users**
> If changing data directory you must also change XMR wallet and address for security reasons [1]
> Read about why it's a good idea to switch to a new data directory every so often.  [2]
>
> [1] https://www.getmonero.org/2018/09/25/a-post-mortum-of-the-burning-bug.html
> [2] https://bisq.wiki/Switching_to_a_new_data_directory

**Comment by w0000000t** [comment]
The amount of text increased, but likewise for information density, I am cool with it

**Comment by w0000000t** [comment]
I am currently testing this, and it didn't show on startup for me, or at least I cannot be 100% sure it didn't show when starting 1.9.3 testing instance with existing data directory with existing XMR accounts as I did close the popups quite shortly thereafter 😅 
But I used the arbitrator instance, which had no existing XMR accounts, to create an XMR account and restart it, and this popup didn't show.

**Comment by ghost** [comment]
The popup moved to `Account` menu:  https://github.com/bisq-network/bisq/pull/6236#issuecomment-1153658999

**Pull request merged**